<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFoldFrameDialog</class>
 <widget class="QDialog" name="AddFoldFrameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Fold Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelFeatureName">
     <property name="text">
      <string>Feature Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="feature_name_input"/>
   </item>
   <item>
    <widget class="QLabel" name="labelItems">
     <property name="text">
      <string>Items</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="items_table">
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Delete</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="add_item_button">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add data to fold frame&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="modelFeatureComboBox">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Link fold to feature&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
